client/src/components/layout/ReviewTile.js

```diff
@@ -1,12 +1,57 @@
-import React from "react";
+import React, { useState, useRef, useEffect } from "react";
 
 const ReviewTile = (props) => {
+  const inputRef = useRef(null);
+  const [inputVisible, setInputVisible] = useState(false);
+  const [comments, setComments] = useState(props.review.comments);
```

Comment:
Setting props in state here will only take effect on the first initial render, we want to avoid this